SUI/SUI/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,45 @@
+// LoginViewModel.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Вью модель экрана логина
+final class LoginViewModel: ObservableObject {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let phoneCountNumber = 18
+        static let emptyString = ""
+        static let minimumPasswordCountString = 5
+        static let maximumPasswordCountString = 16
+    }
+
+    // MARK: - Public Properties
+
+    @Published var loginText = Constants.emptyString
+    @Published var passwordText = Constants.emptyString
+    @Published var isAlertShow = false
+    @Published var isPasswordAlertShow = false
```

Comment:
```suggestion
    @Published var isPasswordAlertShown = false
```
проверь везде что бы было shown больше не буду писать про это